docs/charts/index.yaml

```diff
@@ -3,178 +3,116 @@ entries:
   weblogic-operator:
   - apiVersion: v1
     appVersion: 3.1.0
-<<<<<<< HEAD
```

Comment:
This is a bad merge :(. The easiest fix I've found is to checkout develop, copy the contents of docs/charts to a tmp directory, checkout your branch, copy docs/charts back, and commit. We have a separate JIRA to move the chart generation out of the build step, but we obviously haven't done this yet.

Reply:
I have backed out the changes to docs/charts as you suggested. Let me know if I missed something. Thanks.